BREAKING.md

```diff
@@ -112,6 +113,9 @@ This process is supposed to be a catch-call case for cases where listeners did n
 If possible, it's recommended for DDS event listeners to not throw exceptions, but rather handle them appropriately without involving DDS itself.
 The purpose of this change to ensure that data model stays always synchronized with data projection that event listeners are building. If event listener is not able to fully / correctly process change event, that likely means data synchronization is broken and it's not safe to continue (and potentially, corrupt document).
 
+### Replace `innerRequestHandler` with `rootDataStoreRequestHandler`
```

Comment:
Preferably, our customers don't replace with `rootDataStoreRequestHandler` unless they really need it, so maybe let's downplay that aspect.

```suggestion
### `innerRequestHandler` removed
```